src/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 200px;
+  margin-right: #{$card-right-margin};
+  margin-top: #{$card-top-margin};
+  padding: 32px 16px 16px;
+  color: #060b35;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-child(-n + 4) {
+    margin-top: 0;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code-article {
+    align-self: flex-start;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    font-style: normal;
+    color: #616070;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 16px;
+
+    &-name {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &-value {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    width: 166px;
+    padding: 11px 0;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    text-decoration: none;
+    color: #fff;
+    border-radius: 5px;
+    background-color: #00acdc;
+    border: 1px solid #00acdc;
+
+    &:hover {
+      box-sizing: border-box;
```

Comment:
box-sizing: border-box; add this property to default button, or better to * selector